app/bundles/PointBundle/Entity/LeadPointLog.php

```diff
@@ -31,8 +38,9 @@ public static function loadMetadata(ORM\ClassMetadata $metadata)
     {
         $builder = new ClassMetadataBuilder($metadata);
 
-        $builder->setTable('point_lead_action_log')
-            ->setCustomRepositoryClass('Mautic\PointBundle\Entity\LeadPointLogRepository');
+        $builder->setTable(self::TABLE_NAME)
+            ->setCustomRepositoryClass(LeadPointLogRepository::class)
+            ->addIndex(['internal_id'], 'internal_id');
```

Comment:
If we all agree that the index is not necessary if the internal_id column is part of the primary key then this should be dropped as well.
```suggestion
            ->setCustomRepositoryClass(LeadPointLogRepository::class);
```